lib/jxl/enc_modular.cc

```diff
@@ -1089,7 +1091,14 @@ Status ModularFrameEncoder::PrepareEncoding(ThreadPool* pool,
   tree_ = std::move(decoded_tree);
 
   if (WantDebugOutput(aux_out)) {
-    PrintTree(tree_, aux_out->debug_prefix + "/global_tree");
+    std::ostringstream os;
```

Comment:
I'd use std::to_string() instead but maybe that's just me :)

Reply:
Changed to std::to_string()